networking/configuring-a-custom-pki.adoc

```diff
@@ -40,6 +40,12 @@ The `trustedCA` field is a reference to a `ConfigMap` containing the custom
 certificate and key pair used by the cluster component.
 ====
 
+== Alternatives for additional trust
+
+Cluster components may provide alternative mechanisms for configuring additional trust, for situations when a proxy is not needed:
```

Comment:
I might say  "If you do not use a proxy, you can configure some cluster components to provide additional trust instead:"